lxd/main_checkfeature.go

```diff
@@ -125,12 +131,25 @@ void is_uevent_aware()
 	uevent_aware = true;
 }
 
+#ifndef SECCOMP_RET_USER_NOTIF
+#define SECCOMP_RET_USER_NOTIF 0x7fc00000U
+#endif
+
+void is_seccomp_notify_aware(void)
+{
+	__u32 action[] = { SECCOMP_RET_USER_NOTIF };
+	seccomp_notify_aware = (syscall(__NR_seccomp, SECCOMP_GET_ACTION_AVAIL,
```

Comment:
For some reason it doesn't let me use `seccomp()` directly so I'm going through `syscall()`. Whatever.

Reply:
libc doesn't export a seccomp() :)